portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,21 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+
+ window.addEventListener("scroll", function() {
+
+  var elementTarget = document.getElementById("proj");
```

Comment:
You're including this script on pages other than the index; check the console when viewing other pages, do you get errors?

Reply:
Yes, I shouldn't have copied the header. I took this out.